docs/source/modin/supported/series_str_supported.rst

```diff
@@ -76,7 +76,7 @@ the method in the left column.
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``join``                    | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``len``                     | Y                               |                                                    |
+| ``len``                     | P                               | Only string and list data values are supported.    |
```

Comment:
i think we want to be clear that only string and list columns are supported, because i can have a column with mixed string and list data, but i don't think it work correctly today

Reply:
Added more clarification.